tests/integration-tests/test-suites/eventProcessedByFirehoseAudit.ts

```diff
@@ -0,0 +1,78 @@
+import { invokeLambdaFunction } from '../support/utils/aws/lambda/invokeLambda'
+import { getEnv } from '../support/utils/getEnv'
+import mockSQSEvent from '../support/test-auth-event.json'
+import { randomUUID } from 'crypto'
+import { cloudwatchLogFilters } from '../constants/cloudWatchLogFilters'
+import { checkSplunkBucketForEventId } from '../support/utils/aws/s3/checkSplunkBucketForEventId'
+import { baseEvent } from '../constants/baseEvent'
+import { getAuditEvent } from '../support/utils/aws/s3/returnS3FileContents'
+
+import {
+  NestedKeyOf,
+  obfuscateSpecifiedProps
+} from '../support/utils/obfuscateSpecifiedProps'
+import { AuditEvent } from '../support/types/auditEvent'
+import { SQSEvent } from 'aws-lambda'
+import { logSuccessForEventIdInLogGroup } from '../support/utils/helpers'
+import { filterCloudWatchLogs } from '../support/utils/aws/cloudwatch/cloudWatchGetLogs'
+
+describe('events processed by firehose', () => {
+  let eventId: string
+  let messageId: string
+
+  beforeAll(async () => {
+    eventId = randomUUID()
+    messageId = randomUUID()
+
+    const event: SQSEvent = {
+      Records: [
+        {
+          ...mockSQSEvent.Records[0],
+          messageId: messageId,
+          body: JSON.stringify({
+            ...baseEvent,
+            event_id: eventId
+          })
+        }
+      ]
+    }
+
+    await invokeLambdaFunction(
+      getEnv('AUDIT_MESSAGE_DELIMITER_FUNCTION_NAME'),
+      event
+    )
+    console.log(`Event ID for ${baseEvent.event_name}: ${eventId}`)
+  })
+  // Hits lambda directly - Remove and hit infra firehose function instead
+
+  /*
+    Test logs of everything downstream. First should be correct for 1st lambda
+    Replace below Splunk references with correct Audit components:
+    - The Log group of the Delimiter function (Covered)
+    - The Log group of the Encrypt function (Next Lambda)
+
+    Check if we should be checking the S3 buckets also - Add to template regardless
+    */
+  test('Audit event successfully processed, correct fields are obfuscated for a audit event', async () => {
+    // S3 Delimiter Function Cloudwatch Logs
+    const filterDelimiterLogsByUniqueMessageId = await filterCloudWatchLogs(
+      getEnv('AUDIT_MESSAGE_DELIMITER_LOGS_NAME'),
+      [cloudwatchLogFilters.eventPublished, messageId]
+    )
+    expect(filterDelimiterLogsByUniqueMessageId).not.toEqual([])
+    logSuccessForEventIdInLogGroup(
```

Comment:
Not sure what you're doing here, because the delimiter logs don't say anything about event ids, there's nothing you can actually check in the logs here. Not all our tests *have* to check that there's something in the logs, if there's an actual result to check (like the data being in the S3 bucket as we're doing here), then that's enough.